packages/client/helpers/build.ts

```diff
@@ -127,7 +127,7 @@ function bundleTypeDefinitions(filename: string, outfile: string) {
 
 void build([generatorBuildConfig, nodeRuntimeBuildConfig, browserBuildConfig, edgeRuntimeBuildConfig]).then(() => {
   if (process.env.DEV !== 'true') {
-    bundleTypeDefinitions('declaration/client/src/runtime/index', 'runtime/index')
-    bundleTypeDefinitions('declaration/client/src/runtime/index-browser', 'runtime/index-browser')
+    bundleTypeDefinitions('declaration/runtime/index', 'runtime/index')
```

Comment:
Using `tsc` with `--paths null` means that TS outputs less types and thus changing the output folder structure.